### DISCUSSION TITLE
Primeiro de Abril

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#458] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/primeiro-de-abril/

**Comment by brunoaugusto** [comment]
♪ ♫ Do mundo não se leva nada ♪ ♫

**Comment by one-eyed-willy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/videos/primeiro-de-abril/#comment_1087767317>):

> ♪ ♫ Do mundo não se leva nada ♪ ♫

Vamos sorrir e cantar!

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/videos/primeiro-de-abril/#comment_1087767986>):

> Vamos sorrir e cantar!

Lá, lá, lá, lá, hey!

Lá, lá, lá, lá, hey!

Lá, lá, lá, lá

Lá, lá, lá, lá, lá, lá

**Comment by tali-zorah** [comment]
Que bonitinho as frutas falantes ^^
